### PR TITLE
Article: why lit is 🔥

### PR DESCRIPTION
**Pull request by lucienimmink**
Article: Why lit is 🔥

A brief take on LIT, how it's build, how to use it and what it is all about

### CODE REVIEW

**Comment by lucienimmink** [comment]
As bonus you get the HTTP/3 article in this PR as well :P

**Comment by sanderdejong88** [comment]
Planning to publish on 17/06/2022

File: data/blog/why-lit-is-🔥.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+authors: ['lucien-immink']
+theme: 'blue'
+---
+
+In the fast-changing world of front-end development it's hard to keep up with all the new libraries and frameworks out there that promise to be the next big thing. Frameworks tend to do a lot of heavy lifting but also tend to not agree with each other. As a developer this means that you might have to rewrite code to swap out the old framework for the new. It becomes more of an issue if you must rewrite nearly _all_ of your code if the frameworks can't live together.
+
+![Schema of the elements of a URL](/articles/why-lit-is-🔥/standards.png)
+
+Luckily things are changing (again). With architectural choices like [micro frontends](https://www.isaac.nl/nl/developer-blog/micro-frontends/) it is possible to break up a complex and large code base in logical smaller parts. What if those parts share a few components? Which framework should you use? It should be interoperable and future ready. This is one example where _web components_ shine as you can use those [anywhere](https://caniuse.com/custom-elementsv1) you use HTML.
```
Comment (sanderdejong88) [inline data/blog/why-lit-is-🔥.md]: Can we change this to link to: https://www.iodigital.com/en/insights/technology/micro-frontend-future-frontend-development

At some point all agency websites will be gone an this link would become a 404.

Comment (lucienimmink) [reply data/blog/why-lit-is-🔥.md]: Good insight! I'll update the reference!

**Pull request merged**